spec/fixtures/external_certs/ca_intermed2.crt

```diff
@@ -0,0 +1,76 @@
+Certificate:
+    Data:
+        Version: 3 (0x2)
+        Serial Number: 1 (0x1)
+    Signature Algorithm: sha256WithRSAEncryption
+        Issuer: O=US2, OU=US2 Unit, CN=US2
+        Validity
+            Not Before: Jan 30 23:59:02 2019 GMT
+            Not After : Jan 30 23:59:02 2020 GMT
+        Subject: O=US3, OU=US3 Unit, CN=US3
+        Subject Public Key Info:
```

Comment:
Does nothing really complain about having an empty subject in the CA/intermediate CA?  Weird; I would've expected something to get upset about that. :D

Reply:
I think even the things there ( O and OU ) are only required because the configuration specifies them to be required. The only thing I think is needed is the CN ultimately.